AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/interstitials/InterstitialDemoMenuActivity.java

```diff
@@ -0,0 +1,152 @@
+package com.applovin.enterprise.apps.demoapp.ads.applovin.interstitials;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.applovin.adview.AppLovinInterstitialAd;
+import com.applovin.adview.AppLovinInterstitialAdDialog;
+import com.applovin.enterprise.apps.demoapp.ads.applovin.AdStatusActivity;
+import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuActivity;
+import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuItem;
+import com.applovin.sdk.AppLovinAd;
+import com.applovin.sdk.AppLovinAdClickListener;
+import com.applovin.sdk.AppLovinAdDisplayListener;
+import com.applovin.sdk.AppLovinAdLoadListener;
+import com.applovin.sdk.AppLovinAdSize;
+import com.applovin.sdk.AppLovinAdVideoPlaybackListener;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.enterprise.apps.demoapp.R;
+
+public class InterstitialDemoMenuActivity
+        extends DemoMenuActivity
+{
+    @Override
+    protected DemoMenuItem[] getListViewContents()
+    {
+        DemoMenuItem[] result = {
+                new DemoMenuItem( "Basic Integration", "Quick interstitial ads integration", new Intent( this, InterstitialBasicIntegrationActivity.class ) ),
+                new DemoMenuItem( "Manually loading ad", "Use this for greater control over the ad load process", new Intent( this, InterstitialManualLoadingActivity.class ) ),
+                new DemoMenuItem( "Zone Integration", "Create different user experiences of the same ad type", new Intent( this, InterstitialZoneActivity.class ) )
+        };
+        return result;
+    }
+
+    /**
```

Comment:
Why do we have the `InterstitialManualLoadingActivity` class here? It already exists in another file.